### DISCUSSION TITLE
modified date changes when with note flags

### DISCUSSION THREAD

**Discussion by chanclr** [standardnotes/forum#2554] [General]
since 3.5, changing the note flags (Lock, Pin, etc) changes the date modified timestamp

**Comment by atmoio** [comment]
Hmm, good call. I believe the previous behavior was that it wouldn't change the modified date for locking or protecting, but it should change it for pinning.

**Comment by chanclr** [comment]
Ive tested with 3.4.10 and pinning doesn't change it either. Also happens to be my personal preference
